build.gradle

```diff
@@ -7,13 +7,13 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:1.1.0'
```

Comment:
do we have a doc for tracking which gradle/buildToolsVersion/etc. the library is on?